src/common/dout.h

```diff
@@ -84,14 +84,12 @@ struct is_dynamic<dynamic_marker_t<T>> : public std::true_type {};
   }(cct);								\
 									\
   if (should_gather) {							\
-    static size_t _log_exp_length = 80; 				\
-    ceph::logging::Entry *_dout_e = 					\
-      cct->_log->create_entry(v, sub, &_log_exp_length);		\
+    ceph::logging::Entry _dout_e(v, sub);                               \
```

Comment:
in this change, we trade the overhead of allocating `Entry` on heap for the memcpy